stage1/zig.h

```diff
@@ -540,6 +540,15 @@ typedef ptrdiff_t intptr_t;
     static inline int##w##_t zig_div_floor_i##w(int##w##_t lhs, int##w##_t rhs) { \
         return lhs / rhs + (lhs % rhs != INT##w##_C(0) ? zig_shr_i##w(lhs ^ rhs, UINT8_C(w) - UINT8_C(1)) : INT##w##_C(0)); \
     } \
+\
```

Comment:
You should only change `lib/zig.h`; changes will flow to `stage1/zig.h` the next time `stage1/zig1.wasm` is updated.